### ISSUE TITLE
Model Training + Evaluation - Brain Tumor Detection (Week 6)

### ISSUE THREAD

**Issue by margoglvz** [CareTech-General/general-project-fall25#26]
The current issue is that CareTech would like to build a model to predict Pneumonia from X-ray images.

Here is the dataset we would like you to use:
[Brain MRI Images](https://www.kaggle.com/datasets/navoneel/brain-mri-images-for-brain-tumor-detection/data)

Now that you have made your observations and identified or finished your data preprocessing, next step is **model training**! We will be using **Convolutional Neural Networks** which is what we went over during our previous General Meetings. Through testing, see what types of features or tweaks you can make to your CNN to increase your validation accuracy. 

Downloading the Dataset in Google CoLab
Run this command in a code block to download the dataset in the Google CoLab space:

`import kagglehub`

`# Download latest version`
`path = kagglehub.dataset_download("paultimothymooney/blood-cells")`
`print("Path to dataset files:", path)`
